### PR TITLE
scvim: update url to sbl/scvim

### LINKED ISSUE #2550: Redirect supercollider/scvim submodule to sbl/scvim
Following the discussion in #2472 and supercollider/scvim#7 , and the fact that nobody has stepped up and objected to it, we want to redirect the scvim submodule to the sbl/scvim repo.

The recent bugfix of @carlocapocasa and the fact that the sbl/scvim repo has been more maintained are the main reasons to do this.

There were some people that had quite a few issues with their git repos when the scvim folder was moved into a submodule, so we should discuss the best way to do this before making this change.

### PR DESCRIPTION
**Pull request by gusano**
Fixes #2550 

To update the submodule URL locally:

```
git submodule sync
git submodule update --init --recursive --remote
```

### CODE REVIEW

**Comment by brianlheim** [comment]
I'm all for this change, but for those of us that don't use scvim, what burden would this cause to users exactly? That seems to have been left unelaborated in the original discussion at 2550.

**Comment by gusano** [comment]
> what burden would this cause to users exactly?

I don't think users not using scvim should be concerned.

**Comment by gusano** [comment]
Okay it seems sbl/scvim is missing CMakeLists.txt right now.. Closing this.

**Comment by brianlheim** [comment]
> I don't think users not using scvim should be concerned.

Ah, sorry, let me rephrase that. For those of us who don't have working knowledge of scvim (or just for my sake haha), could you explain what problems would this cause for users who do?

**Comment by gusano** [comment]
@brianlheim No problem, you just have to run the above git commands in order to get the submodule pointing to the new repo.

**Comment by brianlheim** [comment]
Thanks! Okay, that seems a lot less complicated that I was made to believe. This should be included in 3.9 IMO

**Comment by bagong** [comment]
We could just push sbl's repo into the SC-one and merge with master? Maybe it's good to keep SC submodules under control of the SC organization? Maybe Stephen would be happy to work on it inside the SC organization?

**Comment by jamshark70** [comment]
I'd appreciate it if someone could verify that you can cherry-pick from a branch based before the vim change into a branch based after it.

When vim was made into a submodule, a git quirk made it really hard for me to go between my sandbox and master. Nobody saw it coming, but it caused a lot of pain to devs.

Please don't just assume it will be fine. Nobody wants that hassle again.

**Comment by gusano** [comment]
@jamshark70 yeah that was rather painful! That's why I initially proposed to push sbl/scvim to supercollider/scvim but it depends on @sbl and @carlocapocasa 

OTOH modern vim users usually use vim plugins so we could also get rid of the submodule altogether and mention sbl/scvim plugin in README_LINUX ;)

**Comment by sbl** [comment]
> That's why I initially proposed to push sbl/scvim to supercollider/scvim but it depends on @sbl and @carlocapocasa

I'm happy to hand that over to the supercollider community - I already tried to suggest this here:  https://github.com/supercollider/scvim/issues/7 which has never been resolved so far.

**Comment by carlocapocasa** [comment]
@sbl 

> push sbl/scvim to supercollider/scvim

The words "optimal solution" quickly came to mind!

> modern vim users usually use vim plugins

@gusano's I like it! In fact, we should probably do both: Add instructions for pathogen, vundle and similar systems but still upgrade supercollider/scvim.

**Comment by gusano** [comment]
> we should probably do both: Add instructions for pathogen, vundle and similar systems but still upgrade supercollider/scvim

@carlocapocasa Yeah this seems like the best.
I am just no sure about the preferred strategy to push sbl/scvim to supercollider/scvim though..

**Comment by carlocapocasa** [comment]
> preferred strategy to push

I actually had a crazy idea about that, that actually might turn out to be a wonderful idea in retrospect: Merge it.

Since it's a fork it should mostly work. In practice we should do a diff and then try to convince @sbl that the diff is harmless, preferably by reducing it to zero.

**Comment by gusano** [comment]
@carlocapocasa I'll have to fiddle a bit as they have unrelated histories but I'm gonna play with it and open a new PR soon.

**Comment by gusano** [comment]
@sbl @carlocapocasa @dvzrv @blacksound I propose merging `sbl/scvim` into `supercollider/scvim` and remove the submodule to inforce users to install it as a vim plugin. Thoughts?

**Comment by carlocapocasa** [comment]
I think that's definitely a reasonable end result to have in mind, but I think a path there involving the new code in the submodule would be easier on the devs.

**Comment by sbl** [comment]
@gusano I don't mind however you approach it. The code is there and I'm happy if the supercollider org takes it over - how you do it I'll leave to the devs here.

**Comment by gusano** [comment]
> I think a path there involving the new code in the submodule would be easier on the devs

@carlocapocasa OK then I'd vote on keeping it as a submodule, but do not install it anywhere with cmake and let the users/devs do that manually.
